jenkins_pipelines/scripts/maintenance_json_generator.py

```diff
@@ -2,139 +2,143 @@
 import json
 import subprocess
 import sys
-
 import requests
+import xml.etree.ElementTree as ET
+from datetime import date, datetime
+from functools import cache
```

Comment:
Please follow the recommended import order https://peps.python.org/pep-0008/#imports